src/Encryption.php

```diff
@@ -86,6 +88,10 @@ public function setAdapter($adapter)
      */
     public function setKey($key)
     {
+        if ($key === '') {
```

Comment:
It will throw the invalid argument exception when trying to use empty string to be set as a key on `setKey` method call.